test/Microsoft.DocAsCode.Build.Common.Tests/MarkdownReaderTest.cs

```diff
@@ -13,21 +13,52 @@ public class MarkdownReaderTest
         [Fact]
         public void TestReadMarkdownAsOverwrite()
         {
-            const string Content = @"---
+            var content = @"---
 uid: Test
 remarks: Hello
 ---
 
 This is unit test!";
             const string FileName = "ut_ReadMarkdownAsOverwrite.md";
-            File.WriteAllText(FileName, Content);
+            File.WriteAllText(FileName, content);
             var results = MarkdownReader.ReadMarkdownAsOverwrite(Environment.CurrentDirectory, FileName);
             Assert.NotNull(results);
             Assert.Equal(1, results.Count);
             Assert.Equal("Test", results[0].Uid);
             Assert.Equal("Hello", results[0].Metadata["remarks"]);
-            Assert.Equal(@"
-This is unit test!", results[0].Conceptual);
+            Assert.Equal("\r\nThis is unit test!", results[0].Conceptual);
```

Comment:
use @"
" for assertion or use `\r\n` for content definition to keep consistency